geekelo/gft-hackathon-webapp-project
# Milestone: Develop and deploy front-end of Home page and About page

# Things worked on;

###  NAVBAR
- Add initial starting files e.g index.html, style.css, index.js, .gitignore, linters, github workflow and media files
- Add structure for the navbar
- Add mobile style for the navbar
- Add desktop style for the navbar
- Make the navbar responsive
- Add js functionality to display and hide mobile menu


### HOMEPAGE LANDING SECTION
- Add structure for the landing section
- Add style rules for the landing section
- Add structure and style for the top navbar section
- Stick the menu to the top of the window on the scroll


### HOMEPAGE HACKATHON STAGES SECTION
- Add structure for hackathon stages
- Add mobile-friendly styles
- Add desktop-friendly styles
- Add transitions on tiles and button areas


### HOMEPAGE FEATURED JUDGES SECTION
- Add structure for judges section
- Add style for judges section
- Display judges sections dynamically using JS

### HOMEPAGE PARTNERS SECTION
- Add structure for partners section
- Add logo files or images for partners logo
- And styles for the partners section

### FOOTER SECTION
- Add structure for the footer section
- Add files for the footer section
- Add styles for the footer section

### RESPONSIVENESS
- Fix styles errors
- Make styles consistent for mobile and desktop versions


### ABOUT PAGE
- Replicate and modify the homepage navbar to satisfy the design requirement for the about-page navbar section 
- Replicate and modify the homepage landing section to satisfy the design requirement for the about-page landing section
- Replicate and modify the homepage footer section to satisfy the design requirement for the about-page footer section
- Add files and styles for the main or content section of the About page
- Make styles consistent for mobile and desktop versions

### DEPLOYMENT (version 1.0.0)
- Test project for bugs
- Test pages, functions, etc to check responsiveness on mobile and desktop screen sizes
- Verify that all errors are fixed
- Deploy project using Github pages
- Include the LIVE DEMO link in the Readme file
- Update the Readme file with relevant information concerning the project
- Update Documentation file
- Add MIT license

<hr>

### LOAD MORE BUTTON
- Add structure for load more button
- Add style rules for button
- Add functionality using JS to load more cards and view less
- Button loads two cards per click
- Button reduces card to two when view less is triggered

> NO FRAMEWORKS WERE USED
